package.json

```diff
@@ -45,11 +45,12 @@
     "babel-polyfill": "6.5.0",
     "babel-preset-es2015": "6.5.0",
     "gulp": "3.9.1",
+    "gulp-add-src": "^0.2.0",
     "gulp-babel": "6.1.2",
     "gulp-concat": "2.6.0",
     "gulp-plumber": "1.1.0",
     "gulp-util": "^3.0.7",
-    "vinyl-source-stream": "^1.1.0",
-    "gulp-add-src": "^0.2.0"
+    "mocha": "^2.4.5",
```

Comment:
Same as above. I usually install this locally but I can remove if you don't want to include it.